jdaviz/tests/test_utils.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+from jdaviz import utils
+
+
+def test_alpha_index():
+    assert utils.alpha_index(0) == 'a'
+    assert utils.alpha_index(1) == 'b'
+    assert utils.alpha_index(25) == 'z'
+    assert utils.alpha_index(26) == 'aa'
+    assert utils.alpha_index(701) == 'zz'

```

Comment:
This can be done via `pytest.mark.parametrize`.

Reply:
ok, but then I'd need a separate test for the `pytest.raises`, right?  Is that preferred?

Reply:
Well... I think it is easier to extend in the future if you want to add more use cases. Also, when one of them fails, `parametrize` tells you which one.